src/content/tr/fundamentals/performance/optimizing-content-efficiency/eliminate-downloads.md

```diff
@@ -1,39 +1,39 @@
 project_path: /web/_project.yaml
 book_path: /web/fundamentals/_book.yaml
-description: En hizli ve en iyi optimize edilmis kaynak, gönderilmemis kaynaktir. Kaynaklarinizi yakin zamanda denetlediniz mi? Denetlemelisiniz ve bunu, her bir kaynagin daha iyi bir kullanici deneyimi saglamaya yardimci oldugundan emin olmak için periyodik bir sekilde yapmalisiniz.
+description: En hizli ve en iyi optimize edilmiş kaynak, gönderilmemiş kaynaktir. Kaynaklarinizi yakin zamanda denetlediniz mi? Denetlemelisiniz ve bunu, her bir kaynağın daha iyi bir kullanici deneyimi saglamaya yardimci oldugundan emin olmak için periyodik bir şekilde yapmalisiniz.
```

Comment:
Google Translate seems to indicate this change tweaks the wording to: "The fastest and best optimized source is the **unsupervised** source." Unsupervised seems like the wrong word here.

Reply:
Updated.

Reply:
Can you check again?